cluster/saltbase/salt/kube-dns/skydns-rc.yaml.in

```diff
@@ -83,7 +83,7 @@ spec:
           name: dns-tcp-local
           protocol: TCP
       - name: dnsmasq
-        image: gcr.io/google_containers/dnsmasq:1.1
+        image: gcr.io/google_containers/dnsmasq-amd64:1.3
```

Comment:
The skydns-rc.yaml.in file is machine generated from the .base file - this should be kube-dnsmasq as in the .base file.
Just run make in this folder and it will generate the .in and .sed files with the same version bump and dnsmasq name change.